tests/csv/3.0/tumor_mutational_burden.csv

```diff
@@ -0,0 +1,2 @@
+patient,sample,vcf,tbi,variantcaller,annotation
+hcc1395,hcc1395,/Users/monarchy/Projects/Coding/sarek/HCC1395T_vs_HCC1395N.mutect2.filtered_VEP.ann.vcf.gz,/Users/monarchy/Projects/Coding/sarek/HCC1395T_vs_HCC1395N.mutect2.filtered_VEP.ann.vcf.gz.tbi,mutect2,vep
```

Comment:
Hardcoded?

Reply:
ah yes need to remove it again from this. Couldn't test the tmb stuff yet with the small test data but needed actual files.